committeeoversight/settings.py

```diff
@@ -140,3 +140,50 @@
 LOGIN_REDIRECT_URL = '/'
 
 WAGTAIL_SITE_NAME = 'Committee Oversight'
+
+CURRENT_PERMANENT_COMMITTEES = [
+    'House Committee on Agriculture',
+    'House Committee on Appropriations',
+    'House Committee on Armed Services',
+    'House Committee on Budget',
+    'House Committee on Education and the Workforce',
+    'House Committee on Energy and Commerce',
+    'House Committee on Financial Services',
+    'House Committee on Foreign Affairs',
+    'House Committee on Homeland Security',
+    'House Committee on House Administration',
+    'House Committee on Intelligence (Permanent Select)',
+    'House Committee on Judiciary',
+    'House Committee on Natural Resources',
+    'House Committee on Oversight and Government Reform',
+    'House Committee on Rules',
+    'House Committee on Science, Space, and Technology',
+    'House Committee on Small Business',
+    'House Committee on Transportation and Infrastructure',
+    'House Committee on Veterans\' Affairs',
+    'House Committee on Ways and Means',
+    'Senate Committee on Aging',
+    'Senate Committee on Agriculture, Nutrition, and Forestry',
+    'Senate Committee on Appropriations',
+    'Senate Committee on Armed Services',
+    'Senate Committee on Banking, Housing, and Urban Affairs',
+    'Senate Committee on Budget',
+    'Senate Committee on Commerce, Science, and Transportation',
+    'Senate Committee on Energy and Natural Resources',
+    'Senate Committee on Environment and Public Works',
+    'Senate Committee on Finance',
+    'Senate Committee on Foreign Relations',
+    'Senate Committee on Health, Education, Labor, and Pensions',
+    'Senate Committee on Homeland Security and Governmental Affairs',
+    'Senate Committee on Indian Affairs',
+    'Senate Committee on Intelligence',
+    'Senate Committee on Judiciary',
+    'Senate Committee on Rules and Administration',
+    'Senate Committee on Small Business and Entrepreneurship',
+    'Senate Committee on Veterans\' Affairs',
+]
```

Comment:
This is a problem I run into all the time and still don't have a great answer for -- how should you organize "global variables" that should apply across an entire Django project? I used to store them in the settings file like this, and I still think that works fine for most use cases.

However, lately I've moved in the direction of storing them in a dedicated app-level module, as in https://github.com/datamade/just-spaces/blob/a4f109d1aeb69884103eb0b4cb248576983476fa/fobi_custom/plugins/form_elements/fields/types.py. My thinking here is that `settings.py` is intended for project-level configuration values, typically for configuring app behavior. Sometimes global variables are customizing an app, but more often they actually represent static data that is relevant to a specific app, and not necessarily the whole project (where a project is an instantiation of a given app).

Our line between project/app is usually pretty fuzzy, since most of our apps are single-use apps meant to represent a single site. So for the most part I don't think this decision matters much. The one exception is probably if you're using a separate settings file for tests, in which case these values should definitely be in an app-level module to prevent duplication -- but using separate settings files for tests is an anti-pattern that I hope we can move away from anyway.

Reply:
Makes sense, very helpful to hear your thinking on this! I think I'm going to leave these in settings but may change that if there's more global static data to handle in the future.